lib/pymedphys/_version.py

```diff
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
 # This file has been autogenerated by `poetry run pymedphys dev propagate`
 
-version_info = [0, 41, 0, "dev0"]
-__version__ = "0.41.0-dev0"
+version_info = [0, 41, 0]
```

Comment:
this suggests that something needs to change in a configuration file to bump up to 0, 42, 0, "dev0"